src/constant.ts

```diff
@@ -0,0 +1 @@
+export const CONNECT_TIMEOUT_S = 10_000;
```

Comment:
Since this constant contains the default value of timeout, I think `DEFAULT_TIMEOUT` is the perfect name.